marathon-api/cli.py

```diff
@@ -1,37 +1,55 @@
 #!/usr/bin/env python
 
-import krux.logging
-import sys
-import socket
-import json
-from pprint import pprint
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+
+#
+# Third party libraries
+#
+
 from marathon import MarathonClient
 from marathon.models import MarathonApp
+
+#
+# Internal libraries
+#
+
+import krux.logging
 from krux.cli import Application, get_group, get_parser
+import marathonapi
 
 
-class MarathonClientApp(Application):
+class Marathon_cli_app(Application):
 
     def __init__(self):
-        super(MarathonClientApp, self).__init__(name='marathonclientapp', syslog_facility='local0')
+        super(Marathon_cli_app, self).__init__(name='marathon_cli', syslog_facility='local0')
+        self.api = marathonapi.Api()
         self.marathon_host = self.args.host
         self.marathon_port = self.args.port
         self.marathon_list_apps = self.args.list_apps
         self.marathon_config_file = self.args.config_file
+        self.marathon_get_app = self.args.get_app
+        self.marathon_delete = self.args.delete
 
     def add_cli_arguments(self, parser):
-        super(MarathonClientApp, self).add_cli_arguments(parser)
+        """
+        Adds command line arguments, these options can be seen using -h when calling the app.
+        """
+        super(Marathon_cli_app, self).add_cli_arguments(parser)
         group = get_group(parser, self.name)
 
         group.add_argument(
             '--host',
             default='localhost',
-            help='Marthon server host name or IP address. ',
+            help='Marathon server host name or IP address. ',
         )
         group.add_argument(
             '--port',
             default='8080',
```

Comment:
@dcrampton it got changed in our environment, but the actual upstream default for marathon is still 8080, not sure if @kruxErinWillingham wants to change it to match or no